### DISCUSSION TITLE
kiosk monitor layout

### DISCUSSION THREAD

**Discussion by Isanderthul** [guydavis/machinaris#511] [Q&A] [answered]
I have a monitor in my server cabinet, with a pi in kiosk mode, displaying the machinaris ui. 
Is there a way I can edit the UI to adjust the layout so that the Partial Proofs for Pools shows above the Challenges from Harvesters?
Also set the height of the elements.

**Comment by guydavis** [comment, marked as answer]
Sorry, not currently.  Good suggestion though.  I'll create an Issue from this.
